Add site governance process to contributing guidelines (#66)
apple/swift-org-website: CONTRIBUTING.md

# Contributing to Swift.org

:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:

The following is a set of guidelines for contributing to Swift.org on GitHub and its governance process. Use your best judgment, and feel free to propose changes to this document in a pull request.

#### Table Of Contents

* [Code of Conduct](#code-of-conduct)
* [Overview](#overview)
* [Swift Website Workgroup](#swift-website-workgroup)
* [Governance](#governance)
* [Blog Posts](#blog-posts)
* [Community Participation](#community-participation)
* [Making Pull Requests](#making-pull-requests)
* [Writing a Patch](#writing-a-patch)
* [Submitting Bug Reports](#submitting-bug-reports)

## Code of Conduct

> This project and everyone participating in it is governed by the [Swift Code of Conduct](https://www.swift.org/code-of-conduct/). By participating, you are expected to uphold this code. Please report unacceptable behavior to one of the [Swift Core Team](https://swift.org/community/#community-structure).

Reviewer: this is what the what the code of conduct pages says:

```
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a member of the [Swift Core Team](https://swift.org/community/#community-structure) or by flagging the behavior for moderation (e.g., in the Forums), whether you are the target of that behavior or not. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. Maintainers are obligated to maintain confidentiality with regard to the reporter of an incident. The site of the disputed behavior is usually not an acceptable place to discuss moderation decisions, and moderators may move or remove any such discussion.
```

## Overview

The Swift.org website has a number of goals that are defined on the [website overview](http://swift.org/website). This lays out the aims of Swift.org and should be considered when contributing.

> Swift.org is guided by the [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) and they are responsible for the content on the site, the design and all technical aspects.

Reviewer 2: ```suggestion
Swift.org is guided by the [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) and they are responsible for the content on the site, the design, and all technical aspects.
```

See `LICENSE.md` for legal information.

## Swift Website Workgroup

> The [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) is a steering committee comprised of members from Apple and the community. They are responsible for guiding the evolution of Swift.org, whether that be any technical or design changes, or content updates.

Reviewer 2: ```suggestion
The [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) is a steering committee comprised of members from Apple and the community. They are responsible for guiding the evolution of Swift.org, including technical decisions, design changes, and content updates.
```

> The SWWG should be consulted before proposing any substantial changes to Swift.org, either on the forums, opening an issue on GitHub or via [Slack](https://swift-server.slack.com/archives/C03FUTB0NTD) - (you can join [here](https://join.slack.com/t/swift-server/shared_invite/zt-1a3hxb9r5-8sFU3D7JUvaP5QO1AjSivg)).

Reviewer: for these purposes, I think we want folks to reach out by contacting us on the forums rather than ephemeral communication channels like email or slack

## Governance

> Swift.org contributions must adhere to the website's [governance process](https://www.swift.org/website-governance/) which dictate how contributions will be accepted. The process covers different scenarios for contributing and how contributions will be accepted.

Reviewer: dictate -> define

## Code of Conduct

This project and everyone participating in it is governed by the [Swift Code of Conduct](https://www.swift.org/code-of-conduct/). By participating, you are expected to uphold this code. Please report unacceptable behavior to one of the [Swift Core Team](https://swift.org/community/#community-structure) members.

## Overview

The Swift.org website has a number of goals that are defined in the [website overview](http://swift.org/website). This lays out the aims of Swift.org and should be considered when contributing.

Swift.org is guided by the [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) and they are responsible for the content on the site, the design, and all technical aspects.

See `LICENSE.md` for legal information.

## Swift Website Workgroup

The [Swift Website Workgroup (SWWG)](https://www.swift.org/website-workgroup) is a steering committee comprised of members from the community. They are responsible for guiding the evolution of Swift.org, including technical decisions, design changes, and content updates.

> The SWWG should be consulted before proposing any substantial changes to Swift.org, by creating a new post on the forums.

Reviewer 3: It might be good to link to the Swift website forums here.

https://forums.swift.org/c/swift-website/84
